code/modules/mob/living/death.dm

```diff
@@ -35,6 +35,12 @@
 	flick(flick_name, animate)
 
 /mob/living/death(gibbed)
+	if(client && ismob(lastattacker)) //If they were murdered, increase the murderer's client kill count! We CoD now.
+		var/mob/M = lastattacker
+		if(M.client)
+			M.client.murders++ //Your LOVE increased.

```

Comment:
this is false on so many levels.
Hit someone once and then have them killed by the environment = YOU MURDERED THEM!

Even if their death happened 5 hours later while you were logged off.